### DISCUSSION TITLE
How can I change the variable innodb_ft_min_token_size

### DISCUSSION THREAD

**Discussion by OultimoCoder** [planetscale/discussion#364] [Q&A] [answered]
Usually this is a database setting and is dynamic so can't be change on startup. I need it set to 1 so I can do full text search on non whitespace languages like Chinese.

What can I do?

**Comment by orware** [comment, marked as answer]
Hello @OultimoCoder,

I believe we have been chatting about this via ticket over the last several days, but for anyone else viewing this discussion, since values like the `innodb_ft_min_token_size` mentioned above aren't adjustable currently within PlanetScale databases, setting the value to `1` is currently not possible and the request has been shared internally with our team for consideration.

**Reply by frozenBranch** [reply]
Any update on this matter?
